README.md

```diff
@@ -54,8 +54,8 @@ Dapr is currently under community development with preview releases. The master
 Find SDK-specific samples in the links below:
 
 - **[.NET SDK](https://github.com/dapr/dotnet-sdk)**
-  - **[Getting Started with ASP.NET Core](https://github.com/dapr/dotnet-sdk/tree/master/samples/AspNetCore)** - Samples for developing ASP.NET applications using the Dapr .NET SDK
-  - **[Getting Started with .NET Actors](https://github.com/dapr/dotnet-sdk/blob/master/docs/get-started-dapr-actor.md)** - Tutorial for developing actor applications using the Dapr .NET SDK including  **[actor samples](https://github.com/dapr/dotnet-sdk/tree/master/samples/Actor)**
+  - **[Getting Started with ASP.NET Core](https://github.com/dapr/dotnet-sdk/tree/master/examples/AspNetCore)** - Samples for developing ASP.NET applications using the Dapr .NET SDK
+  - **[Getting Started with .NET Actors](https://docs.dapr.io/developing-applications/building-blocks/actors/actors-overview/)** - Tutorial for developing actor applications using the Dapr .NET SDK including  **[actor samples](https://github.com/dapr/dotnet-sdk/tree/master/examples/Actor)**
```

Comment:
Thanks for spotting and fixing this!

So we actually do have a .NET specific actors guide at https://github.com/dapr/dotnet-sdk/blob/master/daprdocs/content/en/dotnet-sdk-docs/dotnet-actors/dotnet-actor-howto.md

No need to correct your PR, however. Another PR fixing these same links came in over the weekend:
#395.

I'll just take #395 and note you as a co-author.